components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -229,75 +232,33 @@ func AzureSchema(machineTypes []string) []byte {
 	return bytes
 }
 
-func GcpTrialSchema() []byte {
-	schema := `{
-          "$schema": "http://json-schema.org/draft-04/schema#",
-          "type": "object",
-          "properties": {
-            "name": {
-              "type": "string"
-            },
-            "region": {
-              "type": "string",
-              "enum": [
-                "europe-west4",
-                "us-east4"
-              ]
-            },
-            "zones": {
-              "type": "array",
-              "items": [
-                {
-                  "type": "string",
-                  "enum": [
-                    "europe-west4-a",
-                    "europe-west4-b",
-                    "europe-west4-c",
-                    "us-east4-a",
-                    "us-east4-b",
-                    "us-east4-c"
-                  ]
-                }
-              ]
-            }
-          },
-          "required": [
-            "name"
-          ]
-        }`
-
-	bytes := []byte(schema)
-	return bytes
-}
-
-func AzureTrialSchema() []byte {
+func TrialSchema() []byte {
 	schema := `{
-          "$schema": "http://json-schema.org/draft-04/schema#",
-          "type": "object",
-          "properties": {
-            "name": {
-              "type": "string"
-            },
-            "region": {
-              "type": "string",
-              "enum": [
-				"eastus",
-				"westeurope"
-              ]
-            },
-            "zones": {
-              "type": "array",
-              "items": [
-                {
-                  "type": "string"
-                }
-              ]
-            }
-          },
-          "required": [
-            "name"
-          ]
-        }`
+  "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Why the schema isn't done similar to `AzureSchema` or `GCPSchema` as `RootSchema` struct?